docs/platform-packet.md

```diff
@@ -40,11 +40,17 @@ retry the boot typically fixes this.
 
 ## Boot
 
-LinuxKit on Packet boots the `kernel+initrd` output from moby
-via
-[iPXE](https://help.packet.net/technical/infrastructure/custom-ipxe). iPXE
-booting requires a HTTP server on which you can store your images. The
-`-base-url` option specifies the URL to the HTTP server.
+LinuxKit on Packet boots the `kernel+initrd` output from moby via
+[iPXE](https://help.packet.net/technical/infrastructure/custom-ipxe)
+which also requires a iPXE script. iPXE booting requires a HTTP server
+on which you can store your images. The `-base-url` option specifies
+the URL to the HTTP server. The boot process expects `<name>-kernel`,
```

Comment:
Maybe this will become clear later but I'm unsure from this whether `base-url` is a local path which will be served by a(n existing) webserver or the public URL from which the directory would be available to others (e.g. packet).

Reply:
rephrased

Reply:
It is clarified in the following examples, so I think it's ok.